### PR TITLE
Fix #8846 : Replace Reviewer and deploy ingestion modal with antd modal in the UI -2

### LINKED ISSUE #8846: UI: Replace all the modal with antd modal in the UI

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
### Describe your changes :

I worked on issue #8846 and Replaced Reviewer and deploy ingestion modal with antd modal in the UI

#
### Type of change :

- [x] Improvement

#
### Frontend Preview (Screenshots) :

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@open-metadata/ui

### CODE REVIEW

**Review by chirag-madlani — APPROVED** [review]

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/AddNewService/postgres.spec.js [hunk 1]
```diff
@@ -124,8 +124,8 @@ it('Add Usage ingestion', () => {
 
   cy.get('[data-testid="view-service-button"]')
     .scrollIntoView()
-    .should('be.visible')
-    .click();
+    .should('exist')
+    .click({ force: true });
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/cypress/e2e/AddNewService/postgres.spec.js:129]: Please avoid exist and forceClick

**Review by ShaileshParmar11 — CHANGES_REQUESTED** [review]

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Glossary.spec.js [hunk 2]
```diff
@@ -514,7 +519,7 @@ describe('Glossary page should work properly', () => {
       .should('be.visible')
       .click();
 
-    cy.get('.tw-modal-container').should('be.visible');
+    cy.get('[data-testid="delete-confirmation-modal"]').should('exist');
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Glossary.spec.js:522]: please update to `be.visible`

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Glossary.spec.js [hunk 3]
```diff
@@ -222,7 +226,7 @@ describe('Glossary page should work properly', () => {
     cy.get('[data-testid="saveAssociatedTag"]').scrollIntoView().click();
     cy.get('[data-testid="glossary-details"]')
       .scrollIntoView()
-      .contains('PersonalData.Personal')
+      .contains('PersonalData')
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Glossary.spec.js:229]: we should category with tag, please revert the change

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Glossary.spec.js [hunk 4]
```diff
@@ -208,10 +211,11 @@ describe('Glossary page should work properly', () => {
     const newDescription = 'Updated description';
     // updating tags
     cy.get('[data-testid="tag-container"]')
-      .scrollIntoView()
-      .should('be.visible')
-      .click();
-
+    .should('exist')
+    .within(() => {
+        cy.get('[data-testid="add-tag"]').should('exist').click()
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Glossary.spec.js:216]: remove exist and use visible

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Glossary.spec.js [hunk 5]
```diff
@@ -175,16 +176,18 @@ describe('Glossary page should work properly', () => {
...
+        .should('exist')
+        .within(() => {
+            cy.contains(NEW_GLOSSARY.name);
+        })
+    });
   });
 
   it('Verify added glossary details', () => {
-    cy.get('[data-testid="glossary-left-panel"]').contains(NEW_GLOSSARY.name).should('be.visible');
+    cy.get('[data-testid="glossary-left-panel"]').should('exist').contains(NEW_GLOSSARY.name);
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Glossary.spec.js:190]: Why we are making this change?

**Review by ShaileshParmar11 — CHANGES_REQUESTED** [review]
Please update all the places
- we must `be.visible` instead of `exist`
- avoid force click

**Review by ShaileshParmar11 — APPROVED** [review]

**Pull request merged**